src/emqtt.erl

```diff
@@ -267,9 +267,7 @@
 
 -type(expire_at() :: non_neg_integer() | infinity). %% in millisecond
 
--type(pendings() :: #{requests := queue:queue(publish_req()),
-                      count := non_neg_integer()
```

Comment:
this count is not needed ?

Reply:
Currently, no. The only thing it was used for: checking if queue is empty, but this is already cheap with `queue:is_empty/1`.